storage/store.go

```diff
@@ -1587,20 +1602,21 @@ func (s *Store) processRaft() {
 					panic(err) // TODO(bdarnell)
 				}
 			}
+			s.processRaftMu.Unlock()
```

Comment:
Yeah. Even in clang, the `ACQUIRED_{BEFORE,AFTER}` annotations described in the linked document are not implemented yet.